space2stats_api/app/settings.py

```diff
@@ -1,15 +1,25 @@
+import os
+
 from pydantic_settings import BaseSettings, SettingsConfigDict
 
 
+current_directory = os.getcwd()    
+env_file_name = 'local_db.env'
+
+if os.path.basename(current_directory) == "space2stats_api":
+    env_path = os.path.join(current_directory, '..', env_file_name)
+else:
+    env_path = os.path.join(current_directory, env_file_name)
+
 class Settings(BaseSettings):
     DB_HOST: str
     DB_PORT: int
     DB_NAME: str
     DB_USER: str
     DB_PASSWORD: str
     DB_TABLE_NAME: str
-
-    model_config = SettingsConfigDict(env_file="../local_db.env")
+        
+    model_config = SettingsConfigDict(env_file=env_path)
```

Comment:
I think it would be fine to just have this to default to a path where we assume that the API will be ran. We usually let this to default to `.env` directly 

using the `current_directory` check is a bit weird IMO.

Reply:
Removing execution of the tests in pre-commit removes the need for this logic also. Removed!